### ISSUE TITLE
bug email

### ISSUE THREAD

**Issue by bruggerk** [usegalaxy-no/galaxyadmin#24]
Currently the email goes to: admin@usegalaxy.no 
should be changed to: contact@bioinfo.no

**Comment by torfinnnome** [comment]
Am I the only one that find it strange that the email address is not contact@elixir.no?

**Comment by ehj000** [comment]
No, this (and nels@bioinfo.no) are on the to-do list of WP4. However, they are not administered by us

**Comment by bruggerk** [comment]
nels@bioinfo.no should be managed by UiB so contact points: kidane/olaf/kjell

how about a contact@usegalaxy.no?

**Comment by ehj000** [comment]
David D will contact the admins. Whats been discussed is contact@elixir.no and nels@elixir.no
